src/handler.js

```diff
@@ -1,12 +1,13 @@
 const crypto = require('crypto');
-const fs = require('fs');
 const path = require('path');
 const os = require('os');
+const stream = require('stream');
 const AWS = require('aws-sdk');
 const axios = require('axios');
 const sharp = require('sharp');
 const Slack = require('slack-node');
 const logger = require('./logger');
+const { Base64Encode } = require('base64-stream');
```

Comment:
What's the value of this over just collecting the chunks up out of interest? I'm sure it probably does more stuff?